scripts/main/lychee.js

```diff
@@ -16,30 +16,124 @@ let lychee = {
 	full_photo: true,
 	downloadable: false,
 	public_photos_hidden: true,
-	share_button_visible: false, // enable only v4+
-	api_V2: false, // enable api_V2
-	sub_albums: false, // enable sub_albums features
-	admin: false, // enable admin mode (multi-user)
-	may_upload: false, // enable possibility to upload (multi-user)
-	is_locked: false, // locked user (multi-user)
+	/**
```

Comment:
this comment can be removed. 
Lychee-front does not support v3 anymore.
idem for the later.

Reply:
I will remove it.